metakb/harvesters/base.py

```diff
@@ -0,0 +1,25 @@
+"""A module for the Harvester base class"""
+
+
+class Harvester:
+    """A base class for content harvesters."""
+
```

Comment:
I wonder if we should have a default init() method to provide a guarantee for Alex's iter_assertion() method below? eg

```suggestion
def __init__(self):
    self.assertions = []

```

Reply:
Added in civic.py too